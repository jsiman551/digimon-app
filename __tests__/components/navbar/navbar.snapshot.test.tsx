import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import Navbar from '@/components/navbar'

jest.mock('next/navigation', () => ({
    useRouter: jest.fn(),
}))

describe('Navbar Snapshot', () => {
    it('matches the snapshot', () => {
        // render component
        const { asFragment } = render(<Navbar />)

        // saved snapshot must match with rendered snapshot
        expect(asFragment()).toMatchSnapshot()
    })
})
