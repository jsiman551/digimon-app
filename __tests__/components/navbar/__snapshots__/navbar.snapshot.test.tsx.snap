// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Snapshot matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="navbar flex-col sm:flex-row"
  >
    <div
      class="flex-1"
    >
      <a
        href="/"
      >
        <img
          alt="Digimon Wiki Logo"
          data-nimg="1"
          decoding="async"
          height="200"
          loading="lazy"
          src="svg/the_digimon_logo.svg"
          style="color: transparent;"
          width="200"
        />
      </a>
    </div>
    <div
      class="flex-none mt-3 sm:mt-0"
    >
      <select
        class="select select-primary w-full max-w-xs mr-4"
      >
        <option
          disabled=""
          selected=""
          value=""
        >
          Buscar Digimon Por:
        </option>
        <option
          value="name"
        >
          Nombre
        </option>
        <option
          value="level"
        >
          Level
        </option>
      </select>
      <label
        class="input input-bordered flex items-center gap-2 input-primary"
      >
        <input
          class="grow"
          disabled=""
          placeholder="Buscar"
          type="text"
        />
        <button
          role="button"
        >
          <svg
            class="h-4 w-4 opacity-70"
            fill="currentColor"
            viewBox="0 0 16 16"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M9.965 11.026a5 5 0 1 1 1.06-1.06l2.755 2.754a.75.75 0 1 1-1.06 1.06l-2.755-2.754ZM10.5 7a3.5 3.5 0 1 1-7 0 3.5 3.5 0 0 1 7 0Z"
              fill-rule="evenodd"
            />
          </svg>
        </button>
      </label>
    </div>
  </div>
</DocumentFragment>
`;
